wormhole-connect/src/components/FooterNavBar.tsx

```diff
@@ -16,11 +16,12 @@ const useStyles = makeStyles()((theme) => ({
     flexDirection: 'row',
     gap: '8px',
     padding: '8px',
-    width: '85%',
+    width: '97%',
```

Comment:
wondering why this uses width instead of being width 100% with a horizontal margin or something.

Reply:
you are right, I set it to 100% with margin auto